homeassistant/components/sensor/alpha_vantage.py

```diff
@@ -31,18 +31,6 @@
 CONF_SYMBOLS = 'symbols'
 CONF_TO = 'to'
 
-DEFAULT_SYMBOL = {
-    CONF_CURRENCY: 'USD',
-    CONF_NAME: 'Google',
-    CONF_SYMBOL: 'GOOGL',
-}
-
-DEFAULT_CURRENCY = {
-    CONF_FROM: 'BTC',
-    CONF_NAME: 'Bitcon',
-    CONF_TO: 'USD',
-}
-
 ICONS = {
```

Comment:
Probably unrelated, but why is there a space in `'TRY': 'mdi: currency-try',` (after `mdi:`)... 🤔

Reply:
Done